test/sqlancer/TestLoggableFactory.java

```diff
@@ -0,0 +1,15 @@
+package sqlancer;
+
+import org.junit.jupiter.api.Test;
+import sqlancer.common.log.SQLLoggableFactory;
+
+public class TestLoggableFactory {
+
+    @Test
+    public void testLogCreateTable() {
+        String query = "CREATE TABLE t1 (c1 INT)";
+        SQLLoggableFactory logger = new SQLLoggableFactory();
+        logger.getQueryForStateToReproduce(query);
```

Comment:
Current test didn't check for any oracles and cannot fail, could you add some assertion to test if a create statement is "loggable"? Maybe check `SQLQueryAdapter.couldAffectSchema` field.

Reply:
Agree. I make such changes.